content.css

```diff
@@ -65,7 +65,7 @@
 }
 
 /* Hide "Trends for you" box */
-.bt--notrends [aria-label*="trending" i] {
+.bt--notrends [aria-labelledby="accessible-list-0"] {
```

Comment:
This is a likely very fragile selector that will match something else as Twitter changes its DOM structure.

Reply:
I changed to another one ;)

Reply:
Hahahaha, but it works only in spanish!!!

I have to search for another one...